core/promise/methods/noop/promise_processor.go

```diff
@@ -80,6 +87,7 @@ func (processor *PromiseProcessor) Start(proposal discovery_dto.ServiceProposal)
 
 // Stop stops processing promises
 func (processor *PromiseProcessor) Stop() error {
+	processor.storage.Close()
```

Comment:
Can we stop storage it `Node.Stop` because this dependency will be needed in more places

Reply:
Added storage stopping to the `di.Shutdown()`